src/Ganeti/HTools/CLI.hs

```diff
@@ -259,6 +261,7 @@ defaultOptions  = Options
   , optNodeSim     = []
   , optNodeTags    = Nothing
   , optSkipNonRedundant = False
+  , optStaticKvmNodeMemory = 4096
```

Comment:
Maybe we could have an inhouse only patch for the value? I think one would need at least 2G of RAM for the base Linux OS. Also the current node memory as reported by the hypervisor backend is just wrong, even if 4G is too much for someone, at least it is a sane value.